Building with closures and this keyword/main.js

```diff
@@ -0,0 +1,65 @@
+const clickMeBtn = document.getElementById('click-me-btn')
+const Person = {
+    name: 'John Doe',
+    age: 30,
+    greet() {
+        console.log(`Hello, my name is ${this.name} and my age is ${this.age}`);
+    }
+}
+
+Person.greet(); // Output: Hello, my name is John Doe and my age is 30
+
+Person.greet.call({name: 'Jane Doe', age: 25}); // Output: Hello, my name is Jane Doe and my age
+
+Person.greet.apply({name: 'Jane Doe', age: 25}); // Output: Hello, my name is Jane Doe and my age
+
+Person.greet.bind({name: 'John Doe', age: 25})(); // Output: Hello, my name is Jane Doe and my age
+
+
+function handleClick() {
+    console.log(this.id)
+    console.log(this.textContent)
+}
+
+
+// const handleClick = (e) => {
+//     console.log(this.id)
```

Comment:
Remove commented code block... its good practice to always cleanup your code before pushing.